### ISSUE TITLE
Tailwind plugin idea

### ISSUE THREAD

**Issue by JeremiahSecrist** [chainlift/liftkit-css#1]
I was wondering if I ported this to a tailwind plugin would you be interested in reviewing it? I know the project is in very early stages (congrats on the success so far!) but let me know if this interests you.

**Comment by garrett-from-chainlift** [comment]
God knows I'd need the help! Yeah let's talk about it. Here's my Calendly

https://calendly.com/chainlift/consultation

𝗚𝗮𝗿𝗿𝗲𝘁𝘁 𝗠𝗮𝗰𝗸
Founder, Chainlift • (714) 584-5371

**Comment by benllshua** [comment]
I agree, this can boost your project, let me know if help is needed, I would love to contribute

**Comment by JeremiahSecrist** [comment]
I've already started some preliminary work for the plugin, im still learning the details of how chainlift works. you can see the work so far [here](https://github.com/JeremiahSecrist/liftkit-css).

**Comment by garrett-from-chainlift** [comment]
Y'all are great, I can clarify how LiftKit works, I've been making docs but
theyre taking 5ever

𝗚𝗮𝗿𝗿𝗲𝘁𝘁 𝗠𝗮𝗰𝗸
Founder, Chainlift • (714) 584-5371

**Comment by JeremiahSecrist** [comment]
when we talk tomorrow that will be a major part of the discussion, plus some of the adjustments to how names work to fit into tailwindcss' naming system.

**Comment by JeremiahSecrist** [comment]
you can find my work in https://github.com/JeremiahSecrist/liftkit-tailwindcss

**Comment by JeremiahSecrist** [comment]
once its stable ill talk with you for migrating it over as a new repo.

**Comment by leonardobetti** [comment]
folks any news? a Tailwind combined with Liftkit would help me a lot

**Comment by JeremiahSecrist** [comment]
> folks any news? a Tailwind combined with Liftkit would help me a lot

steady progress is being made [here](https://github.com/JeremiahSecrist/liftkit-tailwindcss) but trying to get it polished before a full release.
